From c5d43f76fd7c3ccb91f1b75618a9c9e8f202505b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?L=C3=A1szl=C3=B3=20D=C3=A1vid?= <laszlo.david@gmail.com>
Date: Fri, 23 Nov 2012 00:26:22 +0100
Subject: [PATCH 201/283] Fix infinite recursive loop

FindBugs description:

There is an apparent recursive loop at IntProperty.java
in method set(Object, Integer)

This method unconditionally invokes itself. This would seem
to indicate an infinite recursive loop that will result in a stack overflow.

Change-Id: I2f52dd3689198cb948925aa65dd9c95be7888fe7
---
 core/java/android/util/IntProperty.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/util/IntProperty.java b/core/java/android/util/IntProperty.java
index 459d6b2..17977ca 100644
--- a/core/java/android/util/IntProperty.java
+++ b/core/java/android/util/IntProperty.java
@@ -42,7 +42,7 @@ public abstract class IntProperty<T> extends Property<T, Integer> {
 
     @Override
     final public void set(T object, Integer value) {
-        set(object, value.intValue());
+        setValue(object, value.intValue());
     }
 
 }
\ No newline at end of file
-- 
1.8.1.1

